docs/get-started/get-started.md

```diff
@@ -63,7 +63,7 @@ android {
 Then you can access the API key in your app's code like this:
 
 ```kotlin
-Gravatar.initialize(BuildConfig.GRAVATAR_API_KEY)
+Gravatar.apiKey(BuildConfig.GRAVATAR_API_KEY).context(appContext)
```

Comment:
```suggestion
Gravatar.apiKey(BuildConfig.GRAVATAR_API_KEY)
    .context(appContext) // Optional
```

Should we add a comment that the `context` is optional?

Reply:
I would say it should work with any context. But let's change it to prevent issues.

Reply:
🤔 What about: 

```
// Optional but highly encouraged.
``` 

As it provides valuable information for us.

Reply:
👍